packages/block-library/src/gallery/edit.js

```diff
@@ -572,6 +636,7 @@ export default function GalleryEdit( props ) {
 								columns: undefined,
 								imageCrop: true,
 								randomOrder: false,
+								aspectRatio: 'auto',
```

Comment:
This isn't sufficient to reset the images - it resets the control, but not the images in the gallery.